invenio_app_ils/ui/src/invenio_app_ils/forms/components/GroupField.scss

```diff
@@ -0,0 +1,16 @@
+fieldset.form-group {
```

Comment:
When changing backoffice sidebar, I was also about the create specific CSS. But the I have preferred to put stuff in the `site` css file of the theme, in particular to be able to re-use the theme's variables. (e.g. hardcoded error and legend colors).
@kprzerwa did you come up with some best practices for this?